homeassistant/components/onewire/sensor.py

```diff
@@ -234,10 +234,8 @@ def get_entities(config):
         # https://developers.home-assistant.io/docs/creating_platform_code_review/#5-communication-with-devicesservices
         _LOGGER.debug("Initializing using OWFS %s", base_dir)
         _LOGGER.warning(
-            "The OWFS implementation of 1-Wire sensors is deprecated, and should be migrated to OWServer (on localhost:4304)."
-        )
-        _LOGGER.warning(
-            "If migration to OWServer is not feasible on your installation, please raise an issue at https://github.com/home-assistant/core/issues/new?title=Unable%20to%20migrate%20onewire%20from%20OWFS%20to%20OWServer"
+            "The OWFS implementation of 1-Wire sensors is deprecated, and should be migrated to OWServer (on localhost:4304). %s",
```

Comment:
We can remove the comma and formatting argument and just continue the string on the next row.